test/protocol/projects/ape/src/.cache/DappToolsERC20/dappnix/erc20.sol

```diff
@@ -0,0 +1,30 @@
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.4.8;
+
+// Token standard API
+// https://github.com/ethereum/EIPs/issues/20
+
+contract ERC20 {
```

Comment:
This is to test dependencies right? That is why a `.cache` file is being explicitly committed?

Reply:
That is correct